dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task250107RemoveEsReadOnlyMonitorJob.java

```diff
@@ -0,0 +1,66 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+
+import java.util.ArrayList;
+
+public class Task250107RemoveEsReadOnlyMonitorJob implements StartupTask {
+
+
+    private final String JOB_NAME = "EsReadOnlyMonitorJob";
+    private final String TRIGGER_NAME = "trigger29";
+    private final String CRON_TABLE_NAME = "qrtz_excl_cron_triggers";
+    private final String TRIGGER_TABLE_NAME = "qrtz_excl_triggers";
+    private final String JOB_TABLE_NAME = "qrtz_excl_job_details";
+
+    /**
+     * Determines if the task should be forced to run.
+     *
+     * @return true if EsReadOnlyMonitorJob is found, false otherwise.
+     */
+    @Override
+    public boolean forceRun() {
+        String sql=String.format("SELECT * FROM %s WHERE job_name=?", JOB_TABLE_NAME);
+        DotConnect dc=new DotConnect();
+        dc.setSQL(sql);
+
+        dc.addParam(JOB_NAME);
+        try {
+            ArrayList results=dc.loadResults();
+            if (!results.isEmpty()){
```

Comment:
let's use `dc.loadResults()` directly, there is no need of another variable if we're not gonna use it anywhere else